README.md

```diff
@@ -1,7 +1,23 @@
-# micro-forms
+# HTML Forms
 Microverse project
-This project consists of building an HTML document that matches the appearance of mint.com’s signup page  
-You can find the original project specification at: https://www.theodinproject.com/courses/html5-and-css3/lessons/html-forms  
```

Comment:
:+1: 

Still, the readme file can be improved, find some inspiration [here](https://github.com/othneildrew/Best-README-Template) and [here](https://www.udacity.com/course/writing-readmes--ud777)